drivers/sub8_videoray_m5_thruster/nodes/thruster_spin_test.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python
+
+if __name__ == '__main__':
+    import sys
```

Comment:
Fixed/.

It could be, but it's pretty nice and easy to use as it is. Not a good use of his time.